cornedbeef/src/main/java/com/swiftkey/cornedbeef/PunchHoleCoachMark.java

```diff
@@ -320,6 +325,19 @@ public PunchHoleCoachMarkBuilder setHorizontalTranslationDuration(long horizonta
             return this;
         }
 
+        /**
+         * Set the padding (in pixels) for the punch hole around the anchor view.
+         *
+         * A positive value will increase the punch hole's radius by N pixels
```

Comment:
Why pixels rather than DPI?

Reply:
Everything else is in pixels at the moment

Reply:
The public APIs should be DIP I think. Like in CoachMark: https://github.com/TouchType/CornedBeef/blob/efb94635380066672e7d6dc95dcc37186ecd69a5/cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java#L107

Reply:
Although the documentation says pixels!

https://github.com/TouchType/CornedBeef/blob/efb94635380066672e7d6dc95dcc37186ecd69a5/cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java#L365

:confused:

Reply:
Ahh okay, I'll change it to dip and change the documentation for CoachMark